readonly/load_models.py

```diff
@@ -48,8 +70,8 @@ def fetch_projects_from_environment() -> list[dict[str, str]]:
             {
                 "url": git_repo_url,
                 "revision": git_repo_revision,
-                "depth": os.getenv("GIT_DEPTH"),
-                "entrypoint": os.getenv("GIT_ENTRYPOINT"),
+                "depth": int(os.getenv("GIT_DEPTH", "0")),
```

Comment:
```suggestion
                "depth": int(os.getenv("GIT_DEPTH", 0)),
```

Reply:
`key, default and the result are str` (see docstring of getenv)